pkg/snapshot/generator/postgres/data/pg_snapshot_generator.go

```diff
@@ -165,6 +165,10 @@ func (sg *SnapshotGenerator) snapshotTable(ctx context.Context, snapshotID strin
 	if err != nil {
 		return err
 	}
+	// the table is empty
+	if tablePageCount < 0 {
```

Comment:
I wonder what does it mean if the page count is zero? Is it ever possible?

Reply:
Yeah, it means 1 page (they start on 0 for some reason).